app/views/handlers/individual_response.py

```diff
@@ -139,7 +138,15 @@ def handle_get(self):
             self.individual_section_id
         )
 
-        if self._list_item_id:
+        if self._request_args.get("journey") == "remove-person":
+            previous_location_url = url_for(
+                "questionnaire.block",
+                list_name=self._list_name,
+                list_item_id=self._list_item_id,
+                block_id="remove-person",
```

Comment:
I think this should be fetched dynamically, otherwise it will break if the id were to change.